col/src/main/java/vct/col/ast/Node.scala

```diff
@@ -817,6 +848,9 @@ final case class JavaNewClass[G](args: Seq[Expr[G]], typeArgs: Seq[Type[G]], nam
 }
 final case class JavaNewLiteralArray[G](baseType: Type[G], dims: Int, initializer: Expr[G])(implicit val o: Origin) extends JavaExpr[G] with JavaNewLiteralArrayImpl[G]
 final case class JavaNewDefaultArray[G](baseType: Type[G], specifiedDims: Seq[Expr[G]], moreDims: Int)(implicit val o: Origin) extends JavaExpr[G] with JavaNewDefaultArrayImpl[G]
+final case class JavaStringValue[G](data: String, t: Type[G])(implicit val o: Origin) extends JavaExpr[G] with JavaStringValueImpl[G]
+
+final case class JavaClassLiteral[G](cls: Type[G])(implicit val o: Origin) extends JavaExpr[G] with JavaClassLiteralImpl[G]
```

Comment:
JavaStringValue is the ast node for a java string literal. LangJavaToCol encodes it using the interning operation, which is looked up via the `t` member of JavaStringValue.

JavaClassLiteral represents the `.class` bit of `Object.class`; I think it's an old piece of code from the javabip branch, and it can be removed.